src/actions/profile/updateUser.js

```diff
@@ -13,3 +18,16 @@ export const resetUpdateAction = () => dispatch => {
     type: RESET_UPDATE_ACTION
   });
 };
+
+export const switchNotifs = (action, option) => dispatch => {
```

Comment:
Use meaningful variable name

Reply:
> Use meaningful variable name

Actually, this has no use. I'll delete it